test/puppetlabs/puppetdb/cli/services_test.clj

```diff
@@ -29,7 +29,7 @@
 
 (deftest whitelisting
   (testing "should log on reject"
-    (let [wl (fs/temp-file)]
+    (let [wl (fs/temp-file "whitelist-log-reject")]
       (.deleteOnExit wl)
```

Comment:
I'm guessing this is true: https://github.com/Raynes/fs/blob/1.4.5/src/me/raynes/fs.clj#L323